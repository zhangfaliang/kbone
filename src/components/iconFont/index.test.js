import { render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import IconFont from './index';

describe('IconFont', async () => {
  it('base', () => {
    expect.assertions(1);
    const wrapper = render(<IconFont prefixCls={null} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
