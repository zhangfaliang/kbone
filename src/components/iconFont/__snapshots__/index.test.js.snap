// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconFont base 1`] = `
<span
  class="iconfont"
/>
`;
